src/ApiPort.VisualStudio.Common/Analyze/ProjectAnalyzer.cs

```diff
@@ -73,14 +77,18 @@ public class ProjectAnalyzer
                 {
                     targetAssemblies.Add(file);
                 }
+
+                //get referenced NuGetPackages
+                var projectNugetReferences = GetPackageReferences(project);
+                referencedNuGetPackages = referencedNuGetPackages.Union(projectNugetReferences).ToList();
```

Comment:
I expect `HashSet.UnionWith` would be clearer and more performant here.

Reply:
Changed to use HashSet